src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -663,3 +663,210 @@ def handle_qkv(sds_sd, ait_sd, sds_key, ait_keys, dims=None):
+            # new_key = new_key.replace("txt_attn.norm.key_norm.scale", "attn.norm_added_k.weight")
+            converted_state_dict[new_key] = original_state_dict.pop(key)
+
+    def remap_single_blocks(key, converted_state_dict, original_state_dict):
+        block_index = int(key.split(".")[1])
+
+        # Hardcoded for now. Can try to infer from state dict in future
+        inner_dim = 3072
+        mlp_ratio = 4.0
+        mlp_hidden_dim = int(inner_dim * mlp_ratio)
+        split_size = (inner_dim, inner_dim, inner_dim, mlp_hidden_dim)
+
+        if "linear1.lora_A.weight" in key:
+            lora_weight = original_state_dict.pop(f"single_blocks.{block_index}.linear1.lora_A.weight")
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_q.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_k.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_v.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.proj_mlp.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+
+        elif "linear1.lora_B.weight" in key:
+            q, k, v, mlp = torch.split(
+                original_state_dict.pop(f"single_blocks.{block_index}.linear1.lora_B.weight"), split_size, dim=0
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_q.lora_B.weight"] = torch.cat([q])
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_k.lora_B.weight"] = torch.cat([k])
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_v.lora_B.weight"] = torch.cat([v])
+            converted_state_dict[f"single_transformer_blocks.{block_index}.proj_mlp.lora_B.weight"] = torch.cat([mlp])
+
+        elif "linear1.lora_B.bias" in key:
+            q_bias, k_bias, v_bias, mlp_bias = torch.split(
+                original_state_dict.pop(f"single_blocks.{block_index}.linear1.lora_B.bias"), split_size, dim=0
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_q.lora_B.bias"] = torch.cat(
+                [q_bias]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_k.lora_B.bias"] = torch.cat(
+                [k_bias]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_v.lora_B.bias"] = torch.cat(
+                [v_bias]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.proj_mlp.lora_B.bias"] = torch.cat(
+                [mlp_bias]
+            )
+
+        else:
+            new_key = key.replace("modulation.lin", "norm.linear")
```

Comment:
Should this not have `new_key = key.replace("single_blocks", "single_transformer_blocks")`?

Reply:
Yes, thanks for the catch! Fixed in the latest commit. There were some other changes need but now state dict loads without warnings/errors